### ISSUE TITLE
Host angular2 API docs once again.

### ISSUE THREAD

**Issue by chalin** [astashov/dartdocs.org#44]
In follow-up to https://github.com/astashov/dartdocs.org/issues/43#issuecomment-296762554, reverse https://github.com/astashov/dartdocs.org/issues/30.

We can do this now because (1) we are using @source rather than @example directives, and (2) links to docs and examples are absolute rather than relative.

The canonical URL, will remain https://webdev.dartlang.org/angular/api, which will host Angular docs, but _only_ the latest.

cc @kwalrath @kevmoo

**Comment by kwalrath** [comment]
Maybe this will happen automatically now that the package is `angular`, no longer `angular2`?

**Comment by chalin** [comment]
I would believe so, right @astashov? Btw, [angular 4.0.0-alpha](https://pub.dartlang.org/packages/angular/versions/4.0.0-alpha) is out; should we see the docs appear soon?

**Comment by astashov** [comment]
Yeah, probabliy. It failed to generate, is it on pub? https://www.dartdocs.org/documentation/angular/4.0.0-alpha/log.txt

**Comment by chalin** [comment]
Yes, it is on pub. E.g.,
```sh
quickstart > pub upgrade
Resolving dependencies... (5.1s)
+ analyzer 0.30.0+2
+ angular 4.0.0-alpha
+ angular_compiler 0.1.1
+ angular_test 1.0.0-beta+4
...
```
